gsheets/gsheets.go

```diff
@@ -0,0 +1,69 @@
+package gsheets
+
+import (
+	"context"
+	"fmt"
+	"log"
+
+	"google.golang.org/api/option"
+	"google.golang.org/api/sheets/v4"
+)
+
+const (
+	SHEET_ID        = "1wD8zsIcn9vUPmL749MFAreXx8cfaYeqRfFoGuSnJ2Lk"
+	SHEET_NAME      = "replies"
+	credentialsFile = "credentials.json"
+)
+
+const (
+	UrlColumn       = 0
+	PostTypeColumn  = 1 // Games, Cooking, etc
+	RespondedColumn = 2 // Whether the post has been responded to
+)
+
+// NewSheetsClient initializes a Google Sheets API client using the provided credentials.
+func NewSheetsClient(credentialsFile string) (*sheets.Service, error) {
+	ctx := context.Background()
+	client, err := sheets.NewService(ctx, option.WithCredentialsFile(credentialsFile))
+	if err != nil {
+		return nil, fmt.Errorf("unable to create Sheets client: %v", err)
+	}
+	return client, nil
+}
+
+// AppendRow adds a new entry to the specified Google Sheet, formatted with URL, Post Type, and Responded checkbox.
+func AppendRow(sheetID, sheetName, url, postType string, credentialsFile string) error {
+	client, err := NewSheetsClient(credentialsFile)
```

Comment:
Did you consider either defining `AppendRow` on some abstracted version of the sheets client or just having it accept the client as an argument so that we could initialize the client once, in `main.go` rather than every time we insert a row?

Reply:
Great flag, sounds good to me! I wasn't paying attention too much attention on the efficiency, but this is an easy change.